src/components/SlideDeck/SlideDeck.scss

```diff
@@ -1,25 +1,46 @@
 @import "../../variables.scss";
 
 .slide-deck {
-    display: grid;
+    display: flex;
```

Comment:
This feature shouldn't require any style changes. These changes result in existing slide decks rendering differently.